tests/integration/tier1/destructive/firewalld-disabled-ol8/test_firewalld_disabled_ol8.py

```diff
@@ -8,7 +8,6 @@
 FIREWALLD_CONFIG_FILE = "/etc/firewalld/firewalld.conf"
 
 
-@pytest.mark.test_firewalld_disabled
 def test_firewalld_disabled_ol8(shell, convert2rhel):
```

Comment:
What's the reason for removing all the markers? Are they not needed?

Reply:
Yeah, as per documentation it is supposed to be done automatically by pytest via the hook they have: https://docs.pytest.org/en/latest/example/markers.html#automatically-adding-markers-based-on-test-names

Reply:
Yep, it is indeed, as Rodolfo says.
I'm applying the suggestion from #1018 
There is also a way to automatically register the marker for the sake of the run during the `pytest_sessionstart()` test collection, but that I see as unnecessary.
Simply ignoring the markers based on the custom prefix and registering those without one is good enough.